addressfinder.js

```diff
@@ -194,27 +209,64 @@ var fieldsForAddressType = {
   /*
    * We expect BC to remove the class "ExpressCheckoutBlockCollapsed" from
    * the selector when the address fields are replaced. Only when we have
-   * observed thid mutation do we bind an AF widget to the "address_1" field.
+   * observed this mutation do we bind an AF widget to the "address_1" field.
    */
-  var bindWidget = function(addressType, selector, oldValue) {
-    var target = document.querySelector(selector),
-        config = { attributes: true, attributeOldValue: true };
+  var setObserver = function(addressType, elementId, oldValue) {
+    if (window.MutationObserver) {
+
+      /* for modern browsers */
+      var target = document.querySelector("#" + elementId),
+          config = { attributes: true, attributeOldValue: true };
+      var observer = new MutationObserver(function(mutations) {
+        mutations.forEach(function(mutation) {
+          if (mutation.attributeName === "class" && mutation.oldValue.indexOf(oldValue) !== -1) {
+            bindToAddressPanel(addressType, fieldsForAddressType[addressType]["address_1"]);
+            observer.disconnect();
+          }
+        });
+      });
+      observer.observe(target, config);
 
-    var observer = new MutationObserver(function(mutations) {
-      mutations.forEach(function(mutation) {
-        if (mutation.attributeName === "class" && mutation.oldValue.indexOf(oldValue) !== -1) {
+    } else if (window.addEventListener) {
+
+      /* for IE 9 and 10 */
+      var target = document.getElementById(elementId);
+      var listener = function(event) {
+        if (event.attrName.toLowerCase() === "class" && event.prevValue.indexOf(oldValue) !== -1) {
           bindToAddressPanel(addressType, fieldsForAddressType[addressType]["address_1"]);
-          observer.disconnect();
+          target.removeEventListener("DOMAttrModified", listener, false);
         }
-      });
-    });
+      }
+      target.addEventListener("DOMAttrModified", listener, false);
+
+    } else {
+      console.log("AddressFinder Error - please use a more modern browser")
```

Comment:
Apparently old browsers do not have `console.log` so it might need to be a silent fail.